src/robusta/integrations/jira/client.py

```diff
@@ -131,6 +153,133 @@ def _get_default_project_id(self):
+        }
+        logging.debug(f"Transitioning issue {issue_id} to {status_name} status with payload: {payload}")
+        response = self._call_jira_api(url, http_method=HttpMethod.POST, json=payload) or {}
+        logging.debug(f"Transitioned issue with response {response}")
+
+    def comment_issue(self, issue_id, text):
+        endpoint = f"issue/{issue_id}/comment"
+        url = self._get_full_jira_url(endpoint)
+        payload = {
+            "body": {
+                "content": [
+                    {
+                        "content": [
+                            {
+                                "text": text,
+                                "type": "text",
+                            }
+                        ],
+                        "type": "paragraph",
+                    }
+                ],
+                "type": "doc",
+                "version": 1,
+            }
+        }
+        logging.debug(f"Adding comment to issue {issue_id} with payload: {payload}")
+        response = self._call_jira_api(url, http_method=HttpMethod.POST, json=payload) or {}
+        logging.debug(response)
+
+    def create_issue(self, issue_data, issue_attachments=None):
+        endpoint = "issue"
+        url = self._get_full_jira_url(endpoint)
+        payload = {
+            "update": {},
+            "fields": {
+                **issue_data,
+                "issuetype": {"id": str(self.default_issue_type_id)},
+                "project": {"id": str(self.default_project_id)},
+            },
+        }
+        logging.debug(f"Create issue with payload: {payload}")
+        response = self._call_jira_api(url, http_method=HttpMethod.POST, json=payload) or {}
+        logging.debug(response)
+        issue_id = response.get("id")
+        if issue_id and issue_attachments:
+            self.add_attachment(issue_id, issue_attachments)
+
+    def update_issue(self, issue_id, issue_data):
+        summary = self._get_nested_property(issue_data, "summary", "")
+        description = self._get_nested_property(issue_data, "description", "")
+        endpoint = f"issue/{issue_id}"
+        url = self._get_full_jira_url(endpoint)
+
+        payload = {
+            "fields": {"summary": summary, "description": description},
+        }
+
+        logging.debug(f"Update issue '{issue_id}' with payload: {payload}")
+        response = self._call_jira_api(url, http_method=HttpMethod.PUT, json=payload) or {}
+        logging.debug(response)
+
+    def manage_issue(self, issue_data, alert_data, issue_attachments=None):
+        existing_issue = self._get_issue_for_labels(issue_data.get("labels", [])) or []
+        status = self._get_nested_property(alert_data, "status", -1)
+        source = self._get_nested_property(alert_data, "source", None)
+        alert_resolved = status == FindingStatus.RESOLVED
+        is_prometheus_alert = source == FindingSource.PROMETHEUS
+
+        if not is_prometheus_alert:
```

Comment:
@arikalon1 I've slightly updated the logic here as follows:

* If it's not a Prometheus alert, we create the issue on Jira without doing any processing
* Else we do our processing